packages/@sanity/desk-tool/src/pane/DocumentPane.js

```diff
@@ -71,6 +72,11 @@ function isPreviewHotkey(event) {
 
 const isValidationError = marker => marker.type === 'validation' && marker.level === 'error'
 
+const isPermissionError = error =>
+  !!(error && error instanceof ClientError && error.message.match(/permissions/gi))
+
+const isClientError = error => !!(error && error instanceof ClientError)
```

Comment:
I think we should "duck type" this instead of checking `instanceof` - since the client used is the one from `@sanity/base`, it's not guaranteed to be the same. If someone installs `@sanity/client` of a different version in their studio, the check won't match, since it's not a direct dependency. 

I'm thinking it would probably be good enough to check for the correct value in `error.statusCode`, unless there is something I'm overlooking?